app/helpers/recipes_helper.rb

```diff
@@ -0,0 +1,2 @@
+module RecipesHelper
```

Comment:
Why is this module here? Should not we delete it if it does not have any members?

Reply:
It was generated with the controller. I can delete it too. We are not going to use it.

Reply:
Oh, nevermind then. If it is auto-generated, just let it be.